doc/INSTALL-ubuntu.md

```diff
@@ -0,0 +1,49 @@
+# Installation on Ubuntu
+
+Tested on Lubuntu Eoan (19.04) using `bash` as the shell.
+
+1. Install dependencies:
+
+        sudo apt install --install-suggests libsdl-gfx1.2-dev
+        sudo apt install qt4-default coreutils build-essential
+
+2. Install BASS:
+   - Visit <http://www.un4seen.com/> and click "Download"
+     next to the penguin at the top of the page
+   - Unzip the downloaded bass24-linux into a directory of your choice.
+     **Note:** Create the directory and cd into it before unzipping, since the
+     ZIP does not have a directory name.  E.g.,
+
+         $ mkdir bass24 && cd bass24 && unzip ../bass24-linux.zip
+
+   - Copy the files.  Assuming `$rocket` holds the path to your Rocket
+     directory:
+
+         $ mkdir $rocket/examples/{lib,include}
+         $ cp x64/libbass.so $rocket/examples/lib    # or ./libbass.so for 32-bit x86
+         $ cp bass.h $rocket/examples/include
+
+3. Build the editor.  In your Rocket directory, run `make`.  You should then
+   be able to run `./editor/editor &` and get a "Rocket Editor" window with
+   no data.
+
+4. Build the example.  Still in your Rocket directory, run
+
+         $ make examples/example_bass
+
+   This will build the example for communication with the editor.
+
+5. Test the example.  Still with the editor open, run:
+
+         $ cd examples
+         $ export LD_LIBRARY_PATH="$(realpath lib):$LD_LIBRARY_PATH"
+             # ^ This is so the system can find libbass.so
+         $ ./example_bass &
```

Comment:
Perhaps `LD_LIBRARY_PATH="$(realpath lib)" ./example_bass` so we don't suggest contaminating the shell? I also don't think previously set `LD_LIBRARY_PATH` needs to be preserved here...

Reply:
- Happy to put it on the same line --- I broke it into two just to have room for the explanatory comment.
- I would strongly prefer to include the old `LD_LIBRARY_PATH` just in case a user has some weird setup.  OK?  (It could be worse --- `"$(realpath lib)${LD_LIBRARY_PATH:+:}$LD_LIBRARY_PATH}" :D ).